cmd/log-shuttle/main.go

```diff
@@ -66,6 +72,15 @@ func ParseFlags(c shuttle.Config) shuttle.Config {
 
 	flag.Parse()
 
+	if skipHeaders {
+		log.Println("Warning: Use of -skip-headers is deprecated, use -input-format=2 (rfc5424) instead")
```

Comment:
Can we parse input-format differently? Having raw constants "2" is pretty crazy bones. Would be nice to support actually saying -input-format=rfc5424, and add that to the help for the flag!